app-shells/starship/starship-1.11.0.ebuild

```
@@ -0,0 +1,430 @@
+	serde_repr-0.1.9
+	sha-1-0.10.0
+	sha1-0.6.1
+	sha1-0.10.1
+	sha1-asm-0.5.1
+	sha1_smol-1.0.0
+	sha2-0.9.9
+	shadow-rs-0.16.3
+	shared_library-0.1.9
+	shell-words-1.1.0
+	signal-hook-0.1.17
+	signal-hook-0.3.14
+	signal-hook-registry-1.4.0
+	similar-2.2.0
+	siphasher-0.3.10
+	slab-0.4.7
+	smallvec-1.9.0
+	socket2-0.4.6
+	starship-battery-0.7.9
+	static_assertions-1.1.0
+	strsim-0.10.0
+	strum-0.22.0
+	strum_macros-0.22.0
+	syn-1.0.99
+	systemstat-0.2.1
+	tauri-winrt-notification-0.1.0
+	tempfile-3.3.0
+	termcolor-1.1.3
+	terminal_size-0.2.1
+	terminfo-0.7.3
+	termios-0.3.3
+	termtree-0.2.4
+	termwiz-0.15.0
+	thiserror-1.0.35
+	thiserror-impl-1.0.35
+	time-0.1.44
+	time-0.3.14
+	time-macros-0.2.4
+	tinyvec-1.6.0
+	tinyvec_macros-0.1.0
+	toml-0.5.9
+	toml_edit-0.14.4
+	tracing-0.1.36
+	tracing-attributes-0.1.22
+	tracing-core-0.1.29
+	typenum-1.15.0
+	ucd-trie-0.1.4
+	uds_windows-1.0.2
+	uluru-3.0.0
+	unicase-2.6.0
+	unicode-bidi-0.3.8
+	unicode-bom-1.1.4
+	unicode-ident-1.0.3
+	unicode-normalization-0.1.21
+	unicode-segmentation-1.10.0
+	unicode-width-0.1.10
+	unicode-xid-0.2.3
+	uom-0.30.0
+	url-2.2.2
+	urlencoding-2.1.2
+	utf8-width-0.1.6
+	utf8parse-0.2.0
+	vcpkg-0.2.15
+	version_check-0.9.4
+	versions-4.1.0
+	vtparse-0.6.2
+	waker-fn-1.1.0
+	walkdir-2.3.2
+	wasi-0.9.0+wasi-snapshot-preview1
+	wasi-0.10.0+wasi-snapshot-preview1
+	wasi-0.11.0+wasi-snapshot-preview1
+	wasm-bindgen-0.2.82
+	wasm-bindgen-backend-0.2.82
+	wasm-bindgen-macro-0.2.82
+	wasm-bindgen-macro-support-0.2.82
+	wasm-bindgen-shared-0.2.82
+	wepoll-ffi-0.1.2
+	which-4.3.0
+	winapi-0.3.9
+	winapi-i686-pc-windows-gnu-0.4.0
+	winapi-util-0.1.5
+	winapi-x86_64-pc-windows-gnu-0.4.0
+	windows-0.39.0
+	windows-0.40.0
+	windows-0.42.0
+	windows-sys-0.36.1
+	windows_aarch64_gnullvm-0.40.0
+	windows_aarch64_gnullvm-0.42.0
+	windows_aarch64_msvc-0.36.1
+	windows_aarch64_msvc-0.39.0
+	windows_aarch64_msvc-0.40.0
+	windows_aarch64_msvc-0.42.0
+	windows_i686_gnu-0.36.1
+	windows_i686_gnu-0.39.0
+	windows_i686_gnu-0.40.0
+	windows_i686_gnu-0.42.0
+	windows_i686_msvc-0.36.1
+	windows_i686_msvc-0.39.0
+	windows_i686_msvc-0.40.0
+	windows_i686_msvc-0.42.0
+	windows_x86_64_gnu-0.36.1
+	windows_x86_64_gnu-0.39.0
+	windows_x86_64_gnu-0.40.0
+	windows_x86_64_gnu-0.42.0
+	windows_x86_64_gnullvm-0.40.0
+	windows_x86_64_gnullvm-0.42.0
+	windows_x86_64_msvc-0.36.1
+	windows_x86_64_msvc-0.39.0
+	windows_x86_64_msvc-0.40.0
+	windows_x86_64_msvc-0.42.0
+	winres-0.1.12
+	yaml-rust-0.4.5
+	zbus-2.3.2
+	zbus_macros-2.3.2
+	zbus_names-2.2.0
+	zvariant-3.6.0
+	zvariant_derive-3.6.0
+"
+
+inherit cargo
+
+DESCRIPTION="The minimal, blazing-fast, and infinitely customizable prompt for any shell"
+HOMEPAGE="https://starship.rs/"
+SRC_URI="
+	https://github.com/starship/starship/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz
+	$(cargo_crate_uris)
+"
+
+LICENSE="Apache-2.0 BSD BSD-2 CC0-1.0 ISC MIT MPL-2.0 Unicode-DFS-2016 Unlicense WTFPL-2 ZLIB"
+
+SLOT="0"
+KEYWORDS="amd64"
+
+BDEPEND=">=virtual/rust-1.64"
```

Shouldn't this be 1.60? I see that here: https://github.com/starship/starship/blob/v1.11.0/Cargo.toml#L26

```suggestion
BDEPEND=">=virtual/rust-1.60"
```

---

Indeed. By some reason I used min version from the master branch of the starship. o_O